### PR TITLE
Fixes issue #2299

### PR DESCRIPTION
**Pull request by bruno-oliveira**
Renamed command

### CODE REVIEW

File: src/lt/objs/find.cljs [hunk 1]
```diff
@@ -216,7 +216,7 @@
 (object/add-behavior! line-input ::exec-active!)
 
 (cmd/command {:command :goto-line
```
Comment (bruno-oliveira) [inline src/lt/objs/find.cljs]: I will do it right now

Comment (bruno-oliveira) [inline src/lt/objs/find.cljs]: Already changed!

Comment (bruno-oliveira) [inline src/lt/objs/find.cljs]: Ive added 3 new PR containing the changes, one per each file to make sure everything is included... waiting feedback!

**Comment by sbauer322** [comment]
I am closing this PR in favor of the other one (https://github.com/LightTable/LightTable/pull/2306) that has been opened.

**Pull request closed without merging**